pkg/nuclide/file-tree/lib/FileTreeController.js

```diff
@@ -299,6 +300,13 @@ class FileTreeController {
         },
       },
       {
+        label: 'Duplicate',
+        command: 'nuclide-file-tree:duplicate-selection',
+        shouldDisplayForSelectedNodes(nodes) {
+          return nodes.length === 1 && !nodes.some(node => node.getItem().isDirectory());
```

Comment:
Since the first assertion confirms the array is length 1, it can be accessed directly:

```js
return nodes.length === 1 && !nodes[0].getItem().isDirectory();
```